src/Spec2-Adapters-Morphic/SpNotebookTabSelectorMorph.class.st

```diff
@@ -48,11 +48,6 @@ SpNotebookTabSelectorMorph >> removeTabIndex: anInteger [
 	"Remove the tab at the given index and
 	adjust any selected (next or first if was last)."
 
-	self tabs removeAt: anInteger.
-
-	self tabs
-		ifNotEmpty: [ self selectedIndex: (self selectedIndex - 1) \\ self tabs size + 1 ]
-		ifEmpty: [ self selectedIndex: 0 ].
-
+	super removeTabIndex: anInteger.
```

Comment:
super implementation performs initial selection reset (self selectedIndex: 0).
It is required to trigger updates of selection when tab is removed. Otherwise new selected tab with same index as removed one will not be updated.